triggers/tests/daml/Retry.daml

```diff
@@ -11,7 +11,8 @@ import DA.Optional
 
 import Daml.Trigger
 
-retryTrigger = runTrigger Trigger
+retryTrigger : Trigger Int
```

Comment:
It would be nice to also change the `CopyTrigger` and update the docs to no longer mention `runTrigger`. Probably also deserves an entry i nthe release notes.